src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router';
 import Banner from '../../common/Banner';
 import ParentSidebar from '../ParentHome/ParentSidebar';
+import CreateNewStudent from './CreateNewStudent';
 import '../../../styles/ParentStyles/index.less';
 
 const ParentFamilyHome = () => {
+  const [studentInfo, setStudentInfo] = useState(null);
```

Comment:
It's usually better to create an initial state object and set useState to that rather than null. Not sure if you were planning to do that in the future, but regardless null also works, just something to keep in mind.